### ISSUE TITLE
Update System Use Notification

### ISSUE THREAD

**Issue by kgallagher** [forumone/peacecorps-site#379]
While logging in on the admin page I noticed the message in the login box still references 18F:

<i>You are accessing a system operated by 18F, a division of the General Services Administration of the United States Government, on behalf of the United States Peace Corps. Unauthorized use of this system is prohibited and subject to criminal and civil penalties. By accessing this system, you are consenting to monitoring and have no expectation of privacy.<i>

We will need to update this. Here is the control in the SSP that I believe covers this:

System Use Notification (AC-8)
The information system:

A.    Displays to users [Assignment: organization-defined system use notification message or banner] before granting access to the system that provides privacy and security notices consistent with applicable federal laws, Executive Orders, directives, policies, regulations, standards, and guidance and states that:
1.    Users are accessing a U.S. Government information system;
2.    Information system usage may be monitored, recorded, and subject to audit;
3.    Unauthorized use of the information system is prohibited and subject to criminal and civil penalties; and
4.    Use of the information system indicates consent to monitoring and recording;

Can you please update the language to meet those requirements? If you would rather we craft it, please let us know.

Thanks,
Kelley

cc; @jacobgodin

**Comment by defbyte** [comment]
yup, this is in progress on my fork (amongst other 18F references) re: #382
